lua/lvim/core/info.lua

```diff
@@ -112,6 +112,9 @@ local function make_auto_lsp_info(ft)
   end
 
   local supported = lsp_utils.get_supported_servers(ft)
+  if supported == nil then
+    return info_lines
+  end
```

Comment:
I'm not sure this fixes anything, since the function shouldn't return `nil` in the first place

https://github.com/LunarVim/LunarVim/blob/9caa457810a26369f848d4f90f6e0ec84e0048e0/lua/lvim/lsp/utils.lua#L67-L72

Reply:
Ohh sure, my bad.
I was running the stable release not the rolling and there are differences!